### PR TITLE
Adding GetSamplingFrequency to OnlineNnet2FeaturePipelineInfo

### PR DESCRIPTION
**Pull request by hugovbraun**
Adding an easy way to get the frequency associated with the feature config.

### CODE REVIEW

**Comment by naxingyu** [comment]
How about add it in online feature interface?

File: src/online2/online-nnet2-feature-pipeline.cc [hunk 1]
```diff
@@ -241,5 +241,17 @@ BaseFloat OnlineNnet2FeaturePipelineInfo::FrameShiftInSeconds() const {
   }
 }
 
+BaseFloat OnlineNnet2FeaturePipelineInfo::GetModelFrequency() {
```
Comment (danpovey) [inline src/online2/online-nnet2-feature-pipeline.cc]: Please call this GetSamplingFrequency()

**Comment by danpovey** [comment]
mm, I see your point Xingyu, but for now I don't want to make big changes
to that code, I'm mostly focused on newer code, so let's make the change
small.

**Comment by naxingyu** [comment]
I see. Cool.

**Comment by hugovbraun** [comment]
@danpovey updated

**Pull request merged**